### PR TITLE
packer 0.8.6

### PR DESCRIPTION
**Pull request by stuwil**

### CODE REVIEW

File: Library/Formula/packer.rb [hunk 1]
```diff
@@ -199,6 +199,7 @@ def install
     packerpath = buildpath/"src/github.com/mitchellh/packer"
     packerpath.install Dir["{*,.git}"]
     Language::Go.stage_deps resources, buildpath/"src"
+    mkdir_p buildpath/"bin"
```
Comment (DomT4) [inline Library/Formula/packer.rb]: Why is this necessary now?

**Comment by dunn** [comment]
Merged in 44fa0937e6a269d8c6420442ed378d2e2ca41d68, thank you!

File: Library/Formula/packer.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/packer.rb]: Sorry, somehow missed this.

Comment (DomT4) [inline Library/Formula/packer.rb]: No worries, not a big deal at all, was just curious.

Comment (stuwil) [inline Library/Formula/packer.rb]: Didn't spend too much time trying to track it down, but the upstream build script was trying to copy to GOPATH/bin, which didn't exist in the buildpath... not sure when that change came in.
